src/config/default-config.js

```diff
@@ -48,89 +49,119 @@ export default class DefaultConfig {
       GAMES.SIMON
     ];
 
-    /******* LIGHTS ********************/
+    /******* LIGHTS  ********************/
     this.LIGHTS = {
       STRIP_A: '0',
       STRIP_B: '1',
       STRIP_C: '2',
       PERIMETER_STRIP: '3',
-      DISK_LIGHT_STRIP: '4'
+      DISK_LIGHT_STRIP: '4',
       HANDSHAKE_STRIP: '5',
       ART_LIGHTS_STRIP: '6'
-    }
+    };
     this.LIGHTS.GAME_STRIPS = [
       this.LIGHTS.STRIP_A,
       this.LIGHTS.STRIP_B,
-      this.LIGHTS.STRIP_C,
+      this.LIGHTS.STRIP_C
     ];
 
+    // These settings effect the default behaviour of panels outside of
+    // any custom logic in any of the games
+    this.PANEL_DEFAULTS = {
+      ACTIVE_INTENSITY: 100,
+      INACTIVE_INTENSITY: 0
+    };
+
     /******* GAMES CONFIGURATION *******/
 
+    this.HANDSHAKE_GAME = {
+      TRANSITION_OUT_TIME: 4000
```

Comment:
@kintel Please add a comment here describing what exactly the purpose and usage of this setting is